layouts/base.html

```diff
@@ -117,12 +119,20 @@
     <div class="govuk-width-container">
       <div class="govuk-footer__meta">
         <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+          <ul class="govuk-footer__inline-list govuk-!-margin-bottom-5">
+            <li class="govuk-footer__inline-list-item">
+              <a href="<%= @base_url %>/#" class="govuk-link govuk-footer__link">Accessibility</a>
```

Comment:
```suggestion
              <a href="<%= @base_url %>/#" class="govuk-link govuk-footer__link">Accessibility statement</a>
```

Reply:
Agree with this one, it mirrors the rest of GOV!